scripts/release.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
we have this "unofficial bash script mode" in other scripts and we should add it here as well